src/librustc/session/config.rs

```diff
@@ -1331,6 +1331,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "for every macro invocation, print its name and arguments"),
     debug_macros: bool = (false, parse_bool, [TRACKED],
         "emit line numbers debug info inside macros"),
+    generate_arange_section: bool = (true, parse_bool, [UNTRACKED],
```

Comment:
This should be `[TRACKED]` as the setting affects output files and we'd want to rebuild them if the setting changes.

Reply:
OK, I changed it.